src/modules/colorPicker/ColorPickerUI/Controls/ColorPickerControl.xaml.cs

```diff
@@ -199,6 +199,7 @@ private void ShowDetails()
                 CurrentColorButton.BeginAnimation(Button.MarginProperty, moveColor);
                 detailsStackPanel.BeginAnimation(StackPanel.OpacityProperty, opacityAppear);
                 detailsGrid.BeginAnimation(Grid.HeightProperty, resize);
+                CurrentColorButton.IsEnabled = false;
```

Comment:
@niels9001 
Does this hides the tooltip too?

Reply:
A disabled control is not showing a tooltip unless specifically enforced (which we are not doing here)